src/Command.js

```diff
@@ -83,6 +117,11 @@ export default class Command {
     );
   }
 
+  getAvailableOptions() {
```

Comment:
I think we can stick with `getOptions()` if we're using command modules. The only command that currently uses the `otherCommandConfigs` getter (to share `PublishCommand` options) can just as easily import the `builder` object from `PublishCommand` itself and spread it into its own `builder` export.

```js
// ...
import { builder as publishOptions } from "./PublishCommand";

// command export

// describe export

export const builder = {
  ...publishOptions,
};

// handler export

export default class UpdatedCommand extends Command {
// ...
```

(the `otherCommandConfigs` bit is still necessary to read `lerna.json` stuff)